src/app/cup/cup-rating-group-table/cup-rating-group-table.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input } from '@angular/core';
+import { CupRatingGroup } from '@models/v2/cup/cup-rating-group.model';
+
+@Component({
+   selector: 'app-cup-rating-group-table',
+   templateUrl: './cup-rating-group-table.component.html',
+   styleUrls: ['./cup-rating-group-table.component.scss']
+})
+export class CupRatingGroupTableComponent {
+   @Input() cupRatingGroup: CupRatingGroup[];
+   @Input() highlightConfig: { promotion: number[]; possible_promotion: number[]; other_competition: number[] };
+
+   showPositionHighlight(index: number, type: string): boolean {
```

Comment:
after creating a separate component for group-table, we don't need function `showPositionHighlight` in `cup-rating-group`, right?